examples/riot/coap_federated/receiver/main.c

```diff
@@ -1,9 +1,7 @@
 #include "reactor-uc/platform/riot/coap_udp_ip_channel.h"
 #include "reactor-uc/reactor-uc.h"
 
-#ifndef REMOTE_ADDRESS
-#define REMOTE_ADDRESS "fe80::44e5:1bff:fee4:dac8"
-#endif
+#define REMOTE_ADDRESS "fe80::cafe:cafe:cafe:1"
```

Comment:
Since we specify our own IP address in the Makefile, perhaps we should also specify the other IP address there as well, and not in the source file?

Reply:
Fixed